tests/command.deploy.test.js

```diff
@@ -7,18 +7,27 @@ const { generateSiteName, createLiveTestSite } = require('./utils/createLiveTest
 
 const siteName = generateSiteName('netlify-test-deploy-')
 
+const validateContent = async ({ siteUrl, path, content, t }) => {
+  let actualContent
+  try {
+    const response = await fetch(`${siteUrl}${path}`)
+    if (response.ok) {
```

Comment:
Should the test always fail when `response.ok` is not `true`? If so, I am wondering if there should be a `t.true(response.ok)`?
Also wondering about whether the test should always fail when `fetch()` or `response.text()` throws. In which case, t.notThrowsAsync() could be used.

Reply:
Oops, nevermind, I see that `content` is sometimes `undefined` in some of the tests. :+1: